examples/2d/move_sprite.rs

```diff
@@ -1,46 +1,54 @@
-//! Renders a 2D scene containing a single, moving sprite.
+//! Renders a 2D scene in which the *Bevy* logo is rendered as a sprite, moving up and down.
 
 use bevy::prelude::*;
 
 fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
         .add_startup_system(setup)
-        .add_system(sprite_movement)
+        .add_system(update_moveables)
+        .add_system(update_birds)
         .run();
 }
 
+/// A component for any sprite that might move.
```

Comment:
```suggestion
/// A component for any entity that might move.
```

Reply:
I like the noun "sprite" because it fits the branding of the example: move sprites.

How about "A component for anyTHING moving."?

Reply:
I have changed this comment to be:

```rs
/// A component for any moving entity.
```

... and also that above `apply_velocity` so that they're symmetrical:

```rs
/// Update all moving entities: those with a velocity and a transform.
```

Reply:
My first idea was to suggest `anything`, but then I found the phrasing awkward, so I preferred replacing with "entity". But the new comment is great.